tests/orquestra/quantum/wip/operators/_operators_test.py

```diff
@@ -80,13 +80,13 @@ def test_pauliterm_from_list_constructor(self, pauli_term):
         assert "Duplicate" in str(e.value)
 
     def test_pauliterm_from_str_constructor(self, pauli_term):
-        assert PauliTerm.from_str(str(pauli_term)) == pauli_term
+        assert PauliTerm(str(pauli_term)) == pauli_term
```

Comment:
I would also add some test cases of different representations such as placing the coefficient in the argument vs the string, having spaces in the string, etc. And also some more badly formatted string representations such as incorrect bracket placement around the operators. Thoughts?

Reply:
I added more test cases and split them into more test functions for clarity. Let me know wyt :)

Reply:
Tests look great!